client/components/UserSettings/index.jsx

```diff
@@ -20,12 +92,15 @@ class UserSettings extends Component {
 UserSettings.propTypes = {
     isLoggedIn: PropTypes.bool,
     username: PropTypes.string,
-    name: PropTypes.string
+    email: PropTypes.string,
+    ats: PropTypes.array,
+    dispatch: PropTypes.func
 };
 
 const mapStateToProps = state => {
-    const { isLoggedIn, username, name } = state.login;
-    return { isLoggedIn, username, name };
+    const { isLoggedIn, username, email } = state.login;
+    const { ats } = state.ats;
```

Comment:
This is unpacking a property named `ats` from an object value stored at `state.ats`, that is to say that it assumes `state` looks like this: 

```js
{
  ats: {
    ats: ...
  }
}
```

When I log `state.ats`, I see an array. So I think this should be: 

```js
const { ats } = state;
```

**Amendment:** 

As I continued to review, I saw that `state.ats.ats` and `state.cycles.cycles` are currently correct. I think this needs to be fixed so that `ats` are at `state.ats` and `cycles` are at `state.cycles`

Reply:
@rwaldron, I think I would like the state to be `state.ats.names` because I intend to use the `state.ats` object to have more information